src/ts/components/touchcontroloverlay.ts

```diff
@@ -0,0 +1,175 @@
+import { Container, ContainerConfig } from './container';
+import { SmallCenteredPlaybackToggleButton } from './smallcenteredplaybacktogglebutton';
+import { PlayerAPI } from 'bitmovin-player';
+import { UIInstanceManager } from '../uimanager';
+import { EventDispatcher, NoArgs, Event as EDEvent } from '../eventdispatcher';
+import { Timeout } from '../timeout';
+import { HTMLElementWithComponent } from '../dom';
+
+export interface TouchControlOverlayConfig extends ContainerConfig {
+  /**
+   * Specify whether the player should be set to enter fullscreen by clicking on the playback toggle button
+   * when initiating the initial playback.
+   * Default is false.
```

Comment:
```suggestion
   * Default: false
```
just to be consistent with the other default declarations